src/streams.h

```diff
@@ -60,6 +60,7 @@ class OverrideStream
     int GetVersion() const { return nVersion; }
     int GetType() const { return nType; }
     size_t size() const { return stream->size(); }
+    void ignore(size_t size) { return stream->ignore(size); }
```

Comment:
The `ignore()` modifies the state of the stream. It is not `const` in `CDataStream` and `CAutoFile` so this would not compile.

Reply:
It does compile, though (I test if my suggestions compile before making them.)

Reply:
It does compile (I test if my suggestions compile before making them), though arguably const on a void function may not make sense. Consider this suggestion a question.

Reply:
"void f() const makes the function itself const. This only really has meaning for member functions. Making a member function const means that it cannot call any non-const member functions, nor can it change any member variables. It also means that the function can be called via a const object of the class." http://www.cplusplus.com/forum/general/12087/

Reply:
I think it should not be `const` because it modifies the state of the stream. I guess it is not `const` for the same reason in the other stream classes.